src/main/java/cz/mzk/utils/SolrUtils.java

```diff
@@ -9,4 +10,13 @@
     public static void setModify(final SolrInputDocument inputDocument, final String fieldKey, final Object fieldValue) {
         inputDocument.addField(fieldKey, Collections.singletonMap("set", fieldValue));
     }
+
+    public static String wrapQueryStrForEmptyValue(String fieldName) {
```

Comment:
mozna *queryNoFieldValue* je tady lepsi... co myslis?

Reply:
Znie to lepsie, (lepsie vystihuje)